external-crates/move/documentation/book/src/packages.md

```diff
@@ -91,30 +151,114 @@ specified_address = "0xB0B"
+Unlike the package manifest however, the `Move.lock` file is not intended for
+you to edit directly. Processes on the toolchain, like the Move compiler,
+access and edit the file to read and append relevant information to it. You
+also must not move the file from the root, as it needs to be at the same level
+as the `Move.toml` manifest in the package.
+
+If you are using source control for your package, make sure the `Move.lock` file
+is checked in to your repository. This ensures every build of your package is
+an exact replica of the original, and that changes to the build will be
+apparent as changes to the `Move.lock` file.
+
+The `Move.lock` file is a TOML file that currently contains the following
+fields. Note: that other fields may be added to the lock file either in the
+future, or by third-party package package managers as well.
+
+### The `[move]` Section
+
+This section contains the core information needed in the lockfile:
+* The version of the lockfile (needed for backwards compatibility checking, and
+  versioning lockfile changes in the future).
+* The hash of the `Move.toml` file that was used to generate this lock file.
+* The hash of the `Move.lock` file of all dependencies. If no depencies are
+  present, this will be an empty string.
+
+```
+[move]
+version = <string> # Lock file version, used for backwards compatibility checking.
+manifest_digest = <hash> # Sha3-256 hash of the Move.toml file that was used to generate this lock file.
+deps_digest = <hash> # Sha3-256 hash of the Move.lock file of all dependencies. If no depencies are present, this will be an empty string.
+```
+
+### The `[move.dependency]` Section
+
+After the Move compiler resolves each of the dependencies for the package it
+writes the location of the dependency to the Move.lock file. If a dependency
+failed to resolve, the compiler will not write the `Move.lock` file and the
+build fails. If all dependencies resolve, the `Move.lock` file contains the
+locations (local and remote) of all of the package's transitive dependencies.
+These will be stored in the `Move.lock` file in the following format:
+
+```
+...
+
+[[move.package]]
+name = "A"
+source = { git = "https://github.com/b/c.git", subdir = "e/f", rev = "a1b2c3" }
+
+[[move.package]]
+name = "B"
+source = { local = "../local-dep" }
+```
+
+### The `[move.toolchain-version]` Section
+
+As mentioned above, additional fields may be added to the lock file by external
+tools. For example, the Sui package manager adds toolchain version information
+to the lock file that can then be used for on-chain source verification:
+
+```
+...
+
+[move.toolchain-version]
+compiler-version = <string> # The version of the Move compiler used to build the package, e.g. 1.21.0
+edition = <string> # The edition of the Move language used to build the package, e.g. 2024.alpha
+flavor = <string> # The flavor of the Move compiler used to build the package, e.g. "sui"
```

Comment:
```suggestion
compiler-version = <string> # The version of the Move compiler used to build the package, e.g. "1.21.0"
edition = <string> # The edition of the Move language used to build the package, e.g. "2024.alpha"
flavor = <string> # The flavor of the Move compiler used to build the package, e.g. "sui"
```

Since `"sui"` is quoted, let's quote all the values?